orchestration/scripts/create-bq-dataset.sh

```diff
@@ -0,0 +1,13 @@
+set -euo pipefail
+
+# Create a dataset, default expiration of 7 days
+declare -ra BQ_CREATE=(
+  bq
+  --location=US
+  mk
+  --dataset
+  --description=${DESCRIPTION}
+  --default_table_expiration=${EXPIRATION}
+  ${PROJECT}:${DATASET}
+)
+1>&2 ${BQ_CREATE[@]}
```

Comment:
```suggestion
1>&2 "${BQ_CREATE[@]}"
```
Also needed to deal with whitespace properly